test/resources/mocks.ts

```diff
@@ -27,7 +27,7 @@ import * as jwt from 'jsonwebtoken';
 import { FirebaseNamespace } from '../../src/firebase-namespace';
 import { FirebaseServiceInterface } from '../../src/firebase-service';
 import { FirebaseApp, FirebaseAppOptions } from '../../src/firebase-app';
-import { Credential, GoogleOAuthAccessToken } from '../../src/credential/credential-interfaces';
+import { credential as _credential, GoogleOAuthAccessToken } from '../../src/credential/index';
```

Comment:
Just curious, why we use the `_credential` alias here?

Reply:
There's already a `credential` global variable in this file.